idea/tests/org/jetbrains/kotlin/idea/quickfix/CommonIntentionActionsTest.kt

```diff
@@ -168,6 +168,20 @@ class CommonIntentionActionsTest : LightPlatformCodeInsightFixtureTestCase() {
         """.trim().trimMargin(), true)
     }
 
+    fun testRemoveConstructorParameters() {
+        myFixture.configureByText("foo.kt", """
+        |class <caret>Foo(i: Int) {
+        |}
+        """.trim().trimMargin())
+
+        myFixture.launchAction(codeModifications.createAddCallableMemberActions(MethodInsertionInfo.constructorInfo(
+                atCaret<UClass>(myFixture), makeParams())).findWithText("Remove 1st parameter from method 'Foo'"))
```

Comment:
It is not an easy task, because it uses `ChangeMethodSignatureFromUsageFix` from Java, which is hardcoded to "method". It is possible to extend it of course in Kotlin and reimplement functionality. But it sounds like another task.